swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id=None, condition=0, age=0, fabric="Unknown"):
+        super().__init__(id, condition, age)
+        self.fabric = fabric
+
+    def get_category(self):
+        return "Clothing"
```

Comment:
👍 Overriding `get_category` means that no matter what kind of `Item` a variable refers to, it will return the appropriate category value.


This would allow us to use `get_category` in `Item`'s `__str__` to inject the proper name, enabling us to reuse the parent logic later in our own `__str__`.